### ISSUE TITLE
WebWorkers support

### ISSUE THREAD

**Issue by OndrejSpanel** [unicredit/akka.js#101]
The now archived https://github.com/sjrd/scala-js-actors links here. That page is one of the most prominent Google search results when searching for Scala.js web workers. It is unclear to me what is the status of the web workers support in akka.js - is this something trivial, not worth documenting, or is it outside of the scope of the project?

**Comment by andreaTP** [comment]
Hi @OndrejSpanel , thanks for the interest in this project and sorry for the delay!
This repository is all about the cross-compilation of the Akka codebase to Scala.Js, so there is no "battery-included" package using WebWorkers.
Still, making use of the [bindings](https://github.com/akka-js/akka.js_bindings), I've worked on a frontend framework with WebWorkers as first class citizens https://github.com/akka-js/akka.js-dom .

Basically there I'm demonstrating that is "trivial" to build a "distributed application" in the frontend leveraging the Actor model.

Does this answer your question?

**Comment by OndrejSpanel** [comment]
Sounds exciting, I will take a look. If I have any more questions, would you prefer me to ask them here, or somewhere else (gitter?).

Thanks.

**Comment by andreaTP** [comment]
gitter for chats please :-)

**Comment by andreaTP** [comment]
https://gitter.im/akkajs/Lobby

**Comment by andreaTP** [comment]
I close this issue and expect to continue the conversation over the chat.
